libbpf-tools/Makefile

```diff
@@ -57,6 +57,10 @@ $(LIBBPF_OBJ): $(wildcard $(LIBBPF_SRC)/*.[ch]) | $(OUTPUT)/libbpf
 		    INCLUDEDIR= LIBDIR= UAPIDIR=			      \
 		    install
 
+$(OUTPUT)/trace_helpers.o: trace_helpers.c | $(OUTPUT)
+	$(call msg,TRACE-HELPERS,$@)
```

Comment:
@anakryiko It seems not covered (tested after remove this rule), I think `$(OUTPUT)/%.o` rule above only cover `app.o`. Am I missing sth?

Reply:
Alright, I figured out why it didn't work. It's because of extra %.skel.h dependency. We can keep this generic by extracting this dependency and adding it only for $(APPS), like in this diff:
```diff
diff --git a/libbpf-tools/Makefile b/libbpf-tools/Makefile
index cf7f8afe9fe4..393011fe29cf 100644
--- a/libbpf-tools/Makefile
+++ b/libbpf-tools/Makefile
@@ -31,11 +31,13 @@ $(OUTPUT) $(OUTPUT)/libbpf:
        $(call msg,MKDIR,$@)
        $(Q)mkdir -p $@

-$(APPS): %: $(OUTPUT)/%.o $(LIBBPF_OBJ) | $(OUTPUT)
+$(APPS): %: $(OUTPUT)/%.o $(LIBBPF_OBJ) $(OUTPUT)/trace_helpers.o | $(OUTPUT)
        $(call msg,BINARY,$@)
        $(Q)$(CC) $(CFLAGS) $^ -lelf -lz -o $@

-$(OUTPUT)/%.o: %.c $(OUTPUT)/%.skel.h $(wildcard %.h) | $(OUTPUT)
+$(patsubst %,$(OUTPUT)/%.o,$(APPS)): %.o: %.skel.h
+
+$(OUTPUT)/%.o: %.c $(wildcard %.h) | $(OUTPUT)
        $(call msg,CC,$@)
        $(Q)$(CC) $(CFLAGS) $(INCLUDES) -c $(filter %.c,$^) -o $@
```